apps/synapse-portal-framework/src/components/elportal/ELGettingStarted.tsx

```diff
@@ -0,0 +1,162 @@
+// TODO Set padding for mobile and desktop
+
+const ELGettingStarted: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: '#39AC97',
+        display: 'grid',
+        gridTemplateColumns: '25% 25% 25% 25%',
+        position: 'relative',
+        pt: '80px',
+        pb: '80px',
+        pl: '80px',
+        pr: '80px',
+      }}
+    >
+      <Box
+        sx={{
+          flex: '1 1 auto',
+          width: '100%',
+          height: '100%',
+          borderTop: '3px solid #ffffff88',
+          pt: '20px',
+          pb: '20px',
+        }}
+      >
+        <Typography
+          variant="headline1"
+          style={{
+            color: 'white',
+            fontFamily: "'Merriweather', serif",
+            fontWeight: 400,
+            fontSize: '24px',
+          }}
+          sx={{
+            width: '100%',
+          }}
+        >
+          Getting Started
+        </Typography>
+        <Typography
+          variant="body1"
+          style={{
+            color: 'white',
+            fontStyle: 'italic',
+            fontSize: '13px',
+            lineHeight: '20px',
+          }}
+          sx={{
+            maxWidth: '100%',
+            mt: '16px',
+            mb: '16px',
+          }}
+        >
+          We provide all the help you need for navigating the portal and
+          accelerating your research.
+        </Typography>
+        <Button
+          variant="contained"
+          href="https://help.eliteportal.org/help/"
+          style={{
+            fontSize: '14px',
+            fontWeight: 500,
+          }}
+          sx={{
+            backgroundColor: '#39AC97',
+            border: '1px solid white',
+            padding: '6px 24px',
+            boxShadow: 'none',
+            '&:hover': {
+              backgroundColor: darken('#5BA998', 0.05),
+              boxShadow: 'none',
+            },
+          }}
+        >
+          Visit Our Help Section
+        </Button>
+      </Box>
+      <IconSquare
+        iconUrl={exploreIcon}
+        headline="Explore"
+        description="Get the latest data, publications, and news from the Exceptional
+         Longevity Research community"
+      />
+      <IconSquare
+        iconUrl={accessIcon}
+        headline="Access"
+        description="Download data, review metadata, and obtain method summaries for
+          the latest groundbreaking studies."
+      />
+      <IconSquare
+        iconUrl={uncoverIcon}
+        headline="Uncover"
+        description="Analyze and transform the data using translational research tools
+          and AI/ML models. Make a longevity breakthrough. Then repeat."
+      />
+    </Box>
+  )
+}
+
+export default ELGettingStarted
+
+export function IconSquare({ iconUrl, headline, description }) {
+  return (
+    <Box
+      sx={{
+        flex: '1 1 auto',
+        width: '100%',
+        height: '100%',
+        overflow: 'hidden',
+      }}
+    >
```

Comment:
This is not in a flex container, so `flex` isn't doing anything. I also think the hidden overflow may end up being a headache if this layout changes

```suggestion
    <Box
      sx={{
        width: '100%',
        height: '100%',
      }}
    >
```

You could consider making this a grid container or flex container, but what you have now looks fine to me.

If you want the pixel-perfect alignment between sections (which is waaaay easier for Adam to do in Figma than for us to do in CSS), you could probably use [subgrid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid). We have a couple of examples I could point you to if you're interested--it's a fairly advanced (and pretty new) CSS feature

Reply:
Hidden overflow will hide excess content and make it impossible to see all of it as a user. If we have excess content in this component, the layout should accommodate it so it isn't hidden. In other cases (but probably not this component) you may want `overflow: scroll` or `overflow: auto` so a scroll bar appears.

Sometimes `overflow: hidden;` is appropriate, but it doesn't seem right in this case (unless I'm missing something, please let me know if I am)